bread/layout/components/datatable.py

```diff
@@ -139,18 +140,17 @@ def with_toolbar(
         title: Any,
         helper_text: Any = None,
         primary_button: Optional[Button] = None,
-        search_backend: Optional[SearchBackendConfig] = None,
         bulkactions: List[Link] = (),
         pagination_config: Optional[PaginationConfig] = None,
         checkbox_for_bulkaction_name="_selected",
+        search_urlparameter=None,
```

Comment:
maybe let's add typehints for all parameters (since most of them already have it)

Reply:
Right, done.